docs/source/AdministratorGuide/HowTo/multiVO.rst

```diff
@@ -119,8 +119,7 @@ Here define the users part of the "superb_user" group, its DIRAC properties, and
 $HOME/.glite/vomses
 -------------------
 
-DIRAC search for VOMS data in the directory pointed by ``$DIRAC_VOMSES`` variable. 
-If this is not present, the default directory is ``$DIRAC/etc/grid-security/vomses``
+DIRAC search for VOMS data in the directory pointed by ``$X509_VOMSES`` variable (there is no default location).
```

Comment:
There still is the default directory

Reply:
hmm, you mean `/etc/vomses`, I meant that it will not start looking into `$DIRAC/etc/grid-security/vomses`..

Reply:
I mean dirac-configure will put things into $DIRAC/etc/grid-security/vomses for usual dirac installations, so that is kind of the default place to look.

Reply:
clarified and pointed out when this will be dropped